setup.py

```diff
@@ -875,27 +875,27 @@ def swig_sources(self, sources, ext=None):
 
 class my_install(install):
     def run(self):
+        """Custom script we run at the end of installing"""
         install.run(self)
-        # Custom script we run at the end of installing - this is the same script
-        # run by bdist_wininst
-        # This child process won't be able to install the system DLLs until our
-        # process has terminated (as distutils imports win32api!), so we must use
-        # some 'no wait' executor - spawn seems fine!  We pass the PID of this
-        # process so the child will wait for us.
-        # XXX - hmm - a closer look at distutils shows it only uses win32api
-        # if _winreg fails - and this never should.  Need to revisit this!
-        # If self.root has a value, it means we are being "installed" into
-        # some other directory than Python itself (eg, into a temp directory
-        # for bdist_wininst to use) - in which case we must *not* run our
-        # installer
-        if not self.dry_run and not self.root:
+        # If self.root has a value, it means we are being "installed" into some other
+        # directory than Python itself - in which case we must *not* run our installer.
+        # bdist_wininst used to trigger this by using a temp directory.
+        # Is this still a concern ?
+        if self.root:
+            print(
+                f"Not executing post install script when not installing in Python itself (self.root={self.root})"
+            )
```

Comment:
As the comment says, is this still a concern? Or was this only done for the sake of an intermediate step in bdist_wininst ?